src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -926,11 +926,18 @@ bool HMICapabilitiesImpl::load_capabilities_from_file() {
   }
 
   try {
-    Json::Reader reader_;
+    Json::CharReaderBuilder builder;
+    const std::unique_ptr<Json::CharReader> reader_(builder.newCharReader());
```

Comment:
@ydementieiev Who is responsible for object destruction? Are you sure that the builder creates object but not destoys it?

Reply:
Builder has default destructor and it will not destroy the object.